packages/wonder-blocks-form/components/radio.md

```diff
@@ -12,15 +12,14 @@ const styles = StyleSheet.create({
     }
 });
 
-const handleChanged = (checked) => console.log(`clicked on radio, will be checked=${checked.toString()}`);
-const groupName = "group";
+const handleChange = (checked) => console.log(`clicked on radio, will be checked=${checked.toString()}`);
 
-<View style={[styles.row]}>
-    <Radio error={false} checked={false} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio error={false} checked={true} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio error={true} checked={false} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio error={true} checked={true} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio disabled={true} checked={false} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio disabled={true} checked={true} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
+<View style={styles.row}>
+    <Radio error={false} checked={false} style={styles.marginRight} onChange={handleChange} />
```

Comment:
Thanks for updating these too.  Having docs that promote best practices is awesome!